### ISSUE TITLE
Research https://prometheus.io/

### ISSUE THREAD

**Issue by buep** [Praqma/noops#48]
Timeboxed, to answer among other questions:

* is it easy to get started?
* can it monitor both mass scale docker containers, and old school pet servers?

**Comment by hoeghh** [comment]
Yes it can. Let me know if you want a demo (like thursday or friday)

**Comment by buep** [comment]
We would love a demo, but first when @timea gets back to our monitoring project.

**Comment by timea** [comment]
The time has just been extended today for the concourse project until the demo is, which is 18th nov, Friday next week. Decide accordingly if you want to wait.
